src/ResourceManager/Sql/Commands.Sql/help/AzureRM.Sql.md

```diff
@@ -8,7 +8,7 @@ Locale: en-US
 
 # AzureRM.Sql Module
 ## Description
-This topic displays help topics for the [Azure SQL Database](https://docs.microsoft.com/azure/sql-database/) Cmdlets.
```

Comment:
Yes, thank you for catching.  Where is the source for this comment? I'd like to change it at the source so that future re-generation of this file doesn't cause this.